doc/ug/dv_methodology/index.md

```diff
@@ -465,7 +465,39 @@ It is recommended to follow these guidelines when collecting coverage:
 *  Collect all coverage metrics (except toggle based on above bullet) on the DUT and all of its non-pre-verified sub-modules
 *  Treat pre-verified sub-modules as ['black-box'](#integration-testing) in terms of coverage collection
 
-### X-Propagation (Xprop)
+### Unreachable Coverage Analysis
+
+Instead of manually reviewing coverage reports to find unreachable code, we use VCS UNR to generate a UNR exclusion file which lists all the unreachable codes.
+VCS UNR (Unreachability) is a formal solution that determines the unreachable coverage objects automatically from simulation.
```

Comment:
Is it necessary to include a link to VCS UNR page? (if they have one)

Reply:
I google "VCS UNR" but didn't find any official related page. I think it's only available in VCS user guide which isn't released to public.